include/libint2/dfbs_generator.h

```diff
@@ -54,12 +54,16 @@ inline std::vector<Shell> uncontract(const std::vector<Shell> &shells) {
   return primitive_shells;
 }
 
-/// @brief returns \Gamma(x)  of x
+/// @brief returns \Gamma(x)
 inline double gamma_function(const double x) { return std::tgamma(x); }
 
-/// @brief return effective exponent of product of two primitive shells
-/// @param shell1 first shell
-/// @param shell2 second shell
+/// @brief Computes an effective exponent for a product of two primitive
+/// gaussians as as described in J. Chem. Theory Comput. 2021, 17, 6886−6900.
+/// \alpha_{eff} = \left[ \frac{\Gamma(L+2)\Gamma(l_\mu +l_\nu +
```

Comment:
to embed LaTeX into dox you need to use tags `\f$` ... see the doxygen documentation and examples throughout the Libint code